docs/mkdocs/docs/aws.md

```diff
@@ -319,13 +319,16 @@ Workaround for STS Authentication with AWS C++ SDK on Certain Linux Distribution
 
 #### Cause:
```

Comment:
The other 3 sections all explain what the problem is before saying what `Cause`s the problem!